docs/Security_Considerations.md

```diff
@@ -8,10 +8,14 @@ Ethereum dApp developers looking to build on Arbitrum will generally find the ex
 
 - **Different EVM/Soldity behavior**: Arbitrum supports every EVM opcode, which means any Solidity (or Vyper, Yuul, etc.) you write will compile and run on Arbitrum. Certain operations, however, have different behavior on Arbitrum than on Ethereum; for a full list, see [Solidity Support](Solidity_Support.md). Also Arbitrum supports most — but not all — Ethereum [precompiles.](Differences_Overview.md)
 
-- **Timing Assumptions with block numbers and timestamps**: Timing assumptions one may make on layer 1 about `block.timestamp` and `block.number` (i.e., that a new block will be produced every 15 seconds on average) won't hold on Arbitrum; relaying on `block.timestamp` on L2 to measure time should only be done over large scales. For more, see [Block Numbers and Time](Time_in_Arbitrum.md)
+- **Timing Assumptions with block numbers and timestamps**: Timing assumptions one may make on layer 1 about `block.timestamp` and `block.number` (i.e., that a new block will be produced every 15 seconds on average) won't hold on Arbitrum; relaying on `block.timestamp` on L2 to measure time should only be done over large scales. For more, see [Block Numbers and Time](Time_in_Arbitrum.md).
```

Comment:
Maybe say explicitly that timestamps are a better way to measure time on Arbitrum?